ci/setup-build-test-gdk.ps1

```diff
@@ -79,6 +79,10 @@ if ((Test-Path env:TEST_CONFIG) -And ($env:TEST_CONFIG -eq "Native")) {
         $tests[0].tests_path += "+${test_map_path}CI_Nightly/"
         $tests[0].test_results_dir = "Slow" + $tests[0].test_results_dir
 
+        # We run functional spatial tests against Vanilla UE4 with replication graph enabled
+        $tests += [TestSuite]::new($gdk_test_project, "SpatialNetworkingMap", "VanillaTestResultsRepGraph", "${test_map_path}CI_Premerge/+${test_map_path}CI_Nightly/",
+        "$user_gdk_settings", $False, "-ini:Engine:[/Script/OnlineSubsystemUtils.IpNetDriver]:ReplicationDriverClassName=/Script/GDKTestGyms.TestGymsReplicationGraph ${user_cmd_line_args}")
+
```

Comment:
This and the lines below are the only things needed to add a new test suite using a custom setting.
This will change with the new CI system, but should be similarly "easy".

Reply:
at least it will have named arguments in Python